includes/class-fontawesome-config-controller.php

```diff
@@ -116,7 +116,10 @@ public function get_item( $request ) {
 			// for build_item().
 			try {
 				$fa = FontAwesome::reset();
-				$fa->load(
+				$r  = new ReflectionMethod( 'FontAwesome', 'load' );
```

Comment:
And this is how we use Reflection to get around the visibility change of `load()` from `public` to `private`.

Reply:
Naughty... 😈